liquibase-standard/src/main/java/liquibase/changelog/visitor/ValidatingVisitor.java

```diff
@@ -184,7 +184,7 @@ protected void validateChange(ChangeSet changeSet, Database database, Change cha
 
     private boolean areChangeSetAttributesValid(ChangeSet changeSet) {
         boolean authorEmpty = StringUtils.isEmpty(changeSet.getAuthor());
```

Comment:
Should we introduce another validation for the author as well, since it also uses isempty? Maybe in another ticket?

Reply:
I agree. If we apply the same logic for `author`, I think we can remove the strict validation we added some time ago.

Reply:
@abrackx did you make a ticket here?

Reply:
Created DAT-18956!